src/components/Community/Contribute/index.tsx

```diff
@@ -1,25 +1,27 @@
 import React from 'react'
-import PropTypes from 'prop-types'
-
-import { logEvent } from '../../../utils/ga'
 
+import { ICommunitySectionTheme } from '../'
+import LayoutWidthContainer from '../../LayoutWidthContainer'
+import Link from '../../Link'
 import CommunityBlock from '../Block'
 import CommunitySection from '../Section'
+import { logEvent } from '../../../utils/front/ga'
 
-import data from '../data'
-
-import { Button, Item, Items, Wrapper } from '../styles'
+import data from '../data.json'
+import sharedStyles from '../styles.module.css'
 
 const { description, mobileDescription, title } = data.section.contribute
 
-const logPR = () => logEvent('community', 'contribute-pr')
-const logBlogpost = () => logEvent('community', 'contribute-blogpost')
-const logTalk = () => logEvent('community', 'contribute-talk')
-const logAmbassador = () => logEvent('community', 'contribute-ambassador')
+const logPR = (): void => logEvent('community', 'contribute-pr')
+const logBlogpost = (): void => logEvent('community', 'contribute-blogpost')
+const logTalk = (): void => logEvent('community', 'contribute-talk')
+const logAmbassador = (): void => logEvent('community', 'contribute-ambassador')
```

Comment:
IIRC there's an option to not have to mark up every return type, if you get bored of doing that.

Reply:
Yep, but we intentionally enabled this rule to have more strict type checking